dexUtils.js

```diff
@@ -391,9 +391,9 @@ function getMoveDescription(moveId = 0) {
     return description;
 }
 
-function getEvolutionStages(monsno = 0) {
-    const evos = EVOLUTION_MAP[monsno];
-}
+// function getEvolutionStages(monsno = 0) {
+//     const evos = EVOLUTION_MAP[monsno];
+// }
```

Comment:
Remove commented code if it's unused

Reply:
there is a lot of unused code in dexUtils.js - linting is disabled for that tho since you are still working on it. but i would like to enable the rule once you are done refactoring that